crates/re_renderer/src/renderer/depth_cloud.rs

```diff
@@ -0,0 +1,356 @@
+//! Renderer that makes it easy to draw point clouds straight out of depth textures.
+//!
+//! Textures are uploaded just-in-time, no caching.
```

Comment:
I would have expected that uploading the textures it out of scope of the renderer.
For re_viewer that would be happening as part of the image cache anyways? I don't think that stuff should be in here

Reply:
I don't agree: these textures are not textures in the usual sense, they're merely portable buffers that hold data the data for the current frame (and most likely _only_ the current frame).
So, similar to the position and color textures in the point cloud renderer really.

I don't see any reason to take some valuable CPU memory in the image cache when they can be uploaded straight from the data store as needed, and will likely be invalid next frame anyway?

Reply:
I think I'm a bit confused what ends up in the image cache exactly. I thought everything that is logged as an image is there anyways and that's where we create gpu resources for those images usually?
I see that practically speaking the image cache is mostly important when there is a decode that we don't want to repeat and that these raw data blobs shouldn't be in there.

In any case still think this should be a two step thing in order to be able to use the image data more than once in the renderer. We will soon (if not already) need the exact same data in different places e.g. to show the depth map as an image. I.e. what I'm saying is that I do think they are textures in the usual sense since they might be used as such in other places, but you might be right that this doesn't necessarily equate image cache!

Anyways, if this is the only usage pattern then we can keep it this way for now. But I kinda don't see how we won't end up wanting to feed in a texture here because we e.g. also render that same texture with color map in a second space view or hover preview.

Reply:
Talked through this together: we'll keep bypassing everything for now, until we resolve a lot of the issues we have with the ImageCache at the moment (#1462 being one of them).